applications/connectivity_bridge/boards/thingy91_nrf52840_readme.txt

```diff
@@ -15,13 +15,13 @@ COM Ports
 This USB interface exposes two COM ports mapped to the physical UART interfaces between the nRF91 Series and nRF52840 devices.
 When opening these ports manually (without using the LTE Link Monitor), be aware that the USB COM port baud rate selection is applied to the UART.
 
-Bluetooth LE Central UART Service
-=================================
+Bluetooth® LE Central UART Service
```

Comment:
**problem:** unrelated changed, not mentioned in commit message.

Reply:
These changes were added later. Updated the commit messages. Thanks Markus.